pkg/customevents/events.go

```diff
@@ -0,0 +1,78 @@
+package customevents
```

Comment:
eventutil or something else would be more accurate

Reply:
Oh, I see that some event code already exists in the util package. Maybe this should just add to that?

Reply:
Yeah, probably best to move it into `util/events.go`.

Reply:
Done